modules/passkey/test/utils/webauthn.ts

```diff
@@ -145,27 +140,26 @@ export class WebAuthnCredentials {
     // <https://w3c.github.io/webauthn/#dictionary-client-data>
     const clientData = {
       type: 'webauthn.create',
-      challenge: base64UrlEncode(publicKey.challenge).replace(/=*$/, ''),
+      challenge: base64UrlEncode(publicKey.challenge),
       origin: `https://${publicKey.rp.id}`,
     }
 
-    const userVerification = publicKey.userVerification ?? 'preferred'
-    const userVerificationFlag = userVerification === UserVerificationRequirement.required ? 0x04 : 0x01
-
     // <https://w3c.github.io/webauthn/#sctn-attestation>
     const attestationObject = {
-      authData: ethers.getBytes(
-        ethers.solidityPacked(
-          ['bytes32', 'uint8', 'uint32', 'bytes16', 'uint16', 'bytes', 'bytes'],
-          [
-            ethers.sha256(ethers.toUtf8Bytes(publicKey.rp.id)),
-            0x40 + userVerificationFlag, // flags = attested_data + user_present
-            0, // signCount
-            `0x${'42'.repeat(16)}`, // aaguid
-            ethers.dataLength(credential.id),
-            credential.id,
-            credential.cosePublicKey(),
-          ],
+      authData: b2ab(
```

Comment:
This was a small bug that existed before but went unnoticed as the previous version of the CBOR decoding library used in the tests accepted both types here. Essentially, we need the field to be an `ArrayBuffer` and not a `Uint8Array`. We got slightly different versions of the library when installing the dependencies in this package, which are more pedantic than their predecessors, and led to this bug being found.